### DISCUSSION TITLE
osu! crashing on all updates after 2023.305.0

### DISCUSSION THREAD

**Discussion by SherlockWh0** [ppy/osu#23181] [Q&A] [answered]
Hello,

This is a long story, so listen close!

I was attempting to make a beatmap, and I had put a lot of work into it.
Then I saw this video: https://www.youtube.com/watch?v=5Yg9Qo5qFXc&ab_channel=osu%21

I decided to install the update but discovered it started crashing on startup.
Then I noticed something. I had skipped an update.
I tried to install the skipped update, and then the latest one, but that still crashed.
But when I tried the update I was on at the start of this (2023.305.0), it started just fine, except for one error. It went something like this: "Your database is too old for this version. Upgrade to the latest version to access it. (2023.305.0 -> 2023.326.1)"

Now I'm on the latest update and lazer is still crashing.

Please help,

Nintendoisgood.

(Also, should I back up my data?)

**Comment by SherlockWh0** [comment]
also, here are logs:
[database.log](https://github.com/ppy/osu/files/11197150/database.log)
[input.log](https://github.com/ppy/osu/files/11197151/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11197152/legacy-ipc.log)
[performance-draw.log](https://github.com/ppy/osu/files/11197153/performance-draw.log)
[performance-update.log](https://github.com/ppy/osu/files/11197154/performance-update.log)
[performance.log](https://github.com/ppy/osu/files/11197155/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11197156/runtime.log)

**Comment by peppy** [comment, marked as answer]
Please install the latest version, you're still behind.

**Comment by SherlockWh0** [comment]
I tried the latest update, it still crashes

**Reply by peppy** [reply]
Please provide logs again if you can.

**Comment by SherlockWh0** [comment]
Here they are:

[database.log](https://github.com/ppy/osu/files/11239477/database.log)
[input.log](https://github.com/ppy/osu/files/11239478/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11239479/legacy-ipc.log)
[performance-draw.log](https://github.com/ppy/osu/files/11239480/performance-draw.log)
[performance-update.log](https://github.com/ppy/osu/files/11239481/performance-update.log)
[performance.log](https://github.com/ppy/osu/files/11239482/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11239483/runtime.log)

**Reply by peppy** [reply]
I can no longer see a crash in your logs. can you take a screenshot of the crash

**Comment by SherlockWh0** [comment]
A video:

https://user-images.githubusercontent.com/107404453/232280078-29ef9e91-c43f-4488-8250-4855c84feb17.mp4

Crash data (from dialog box)
[crash.txt](https://github.com/ppy/osu/files/11241501/crash.txt)

And new logs:

[database.log](https://github.com/ppy/osu/files/11241502/database.log)
[input.log](https://github.com/ppy/osu/files/11241503/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11241504/legacy-ipc.log)
[performance-draw.log](https://github.com/ppy/osu/files/11241505/performance-draw.log)
[performance-update.log](https://github.com/ppy/osu/files/11241506/performance-update.log)
[performance.log](https://github.com/ppy/osu/files/11241507/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11241508/runtime.log)

**Comment by SherlockWh0** [comment]
I tried the latest update, it doesn't work

**Comment by peppy** [comment]
This looks to be the same as #23164. Wondering if we somehow broke something in the process?
